tests/integration/runner

```diff
@@ -135,6 +136,43 @@ def check_args_and_update_paths(args):
         )
 
 
+def check_iptables_legacy():
+    iptables_path = shutil.which("iptables")
+    ip6tables_path = shutil.which("ip6tables")
+
+    if iptables_path is None:
+        print("Error: 'iptables' not found in PATH")
+        sys.exit(1)
+    if ip6tables_path is None:
+        print("Error: 'ip6tables' not found in PATH, ignoring")
+
+    try:
+        file_info = os.stat(iptables_path)
+        file_info_str = str(file_info)
+
+        if "legacy" in file_info_str:
+            print(
+                "iptables is in 'legacy' mode. This is not supported. Please switch to 'nftables' mode."
+            )
+            sys.exit(1)
+
+        if not ip6tables_path:
+            return
+
+        file_info = os.stat(ip6tables_path)
+        file_info_str = str(file_info)
+
+        if "legacy" in file_info_str:
+            print(
+                "ip6tables is in 'legacy' mode. This is not supported. Please switch to 'nftables' mode."
```

Comment:
Can we provide command to use or at least a link to learn how to switch?

Reply:
I can add `consult your distribution manual`, will this be sufficient for you?

Reply:
Actually not as we run docker in docker. You better say that this should be configured on host system... 
It was not trivial and I'm not sure I made it correctly:
https://www.danami.com/clients/knowledgebase/194/How-can-I-switch-to-iptables-nft-from-nftables-after-upgrading-Debian-or-Ubuntu.html?language=english

Reply:
:ok_hand: